setup.py

```diff
@@ -1,16 +1,17 @@
 from setuptools import find_packages, setup
 
 setup(
-    name='mozilla-taar',
-    use_scm_version=True,
+    name='mozilla-taar2',
+    use_scm_version=False,
```

Comment:
Why is this changing? Are you planning on changing the way things are released? If so, why? Was this discussed somewhere?

Reply:
I need to rename the package to upload it into PyPI.  This is a requirement for deploying taar-api.  The existing mozilla-taar repo is still owned by mauro and I can't upload a new version.

Reply:
I mean the `use_scm_version` part.

Reply:
@crankycoder ^

Reply:
I can't get a build to run if `use_scm_version` is on a PR as the build won't run since there's no tag available yet.